newm-server/src/main/kotlin/io/newm/server/features/arweave/repo/ArweaveRepositoryImpl.kt

```diff
@@ -0,0 +1,57 @@
+package io.newm.server.features.arweave.repo
+
+import co.upvest.arweave4s.adt.Signed
+import co.upvest.arweave4s.adt.Transaction
+import co.upvest.arweave4s.adt.Wallet
+import com.softwaremill.sttp.Response
+import com.softwaremill.sttp.SttpBackendOptions
+import com.softwaremill.sttp.Uri
+import com.softwaremill.sttp.asynchttpclient.future.AsyncHttpClientFutureBackend
+import io.ktor.server.application.ApplicationEnvironment
+import io.newm.server.ktx.getSecureConfigString
+import io.newm.shared.koin.inject
+import io.newm.shared.ktx.getConfigString
+import org.koin.core.parameter.parametersOf
+import org.slf4j.Logger
+import scala.Function1
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.io.Source
+import scala.runtime.BoxedUnit
+import java.util.concurrent.Executors
+import co.upvest.arweave4s.adt.`Wallet$`.`MODULE$` as walletApi
+import co.upvest.arweave4s.api.`package`.`Config$`.`MODULE$` as configApi
+import co.upvest.arweave4s.api.tx as txApi
+import co.upvest.arweave4s.api.`package`.`future$`.`MODULE$` as future
```

Comment:
What do we need to do to compile this on the IDE dev environment?

Reply:
No. It's just another .jar file that's included as a dependency. We just have to import some of the mucked-up scala names to make the underlying code work and do the interop between kotlin/scala.